### DISCUSSION TITLE
Max particles and dimensions

### DISCUSSION THREAD

**Discussion by jesusmontera** [quantum-visualizations/qmsolve#13] [Q&A]
I want to simulate the harmonic oscillator with 2 particles in 2 space dimensions, or just 2 particles in 2D, it's possible? I got this error :<BR> kron() takes from 2 to 3 positional arguments but 4 were given

**Comment by rafael-fuente** [comment]
it's not implemented. The reason is that it's because the problem is very computationally expensive (we need a 4D configuration space) and any discretization of the Hamiltonian performed on a personal computer, won't be accurate enough. 

The only way to estimate the problem's solution is by assuming the particles are weakly interacting and going with an orbital-like approximation. It's well known that the N-body quantum mechanical problem is unsolvable without approximations.

Note: If your two particles have only a harmonic-like interaction in free space, then luckily, your problem is separable, and you can solve it as a single-particle Hamiltonian: [https://rjp.nipne.ro/2008_53_9-10/1171_1177.pdf](https://rjp.nipne.ro/2008_53_9-10/1171_1177.pdf)
